.github/workflows/run-test-sh.yml

```
@@ -0,0 +1,39 @@
+name: run-test-shell
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable
+
+  push:
+    branches:
+      - master
+      - stable
+
+jobs:
+  main:
+    name: Build and test image
+    strategy:
+      fail-fast: false # Finish other jobs on failure
+      matrix:
+        include:
+          # - { version: dmd-nightly, exec: dmd   }
+          - { version: dmd-beta,    exec: dmd   }
+          - { version: dmd,         exec: dmd   }
+          - { version: ldc-beta,    exec: ldmd2 }
```

```suggestion
          - { version: ldc-master,    exec: ldmd2 }
```

IMO it's a much better coverage than the infrequent beta. I would advise this for `dmd` too if master wasn't broken.

---

Probably but I would rather not deviate that much from the Travis setup during this transition.
Such enhancements should be made as a seperate PR.

---

> it's a much better coverage than the infrequent beta

It's actually not about test coverage, but about what options are available in the drop down next to the run button (and obviously in the exec rest api). In general the more options the better... as long as they all work :D